cryptography/src/secp256r1.rs

```diff
@@ -200,7 +212,7 @@ mod tests {
     fn test_public_key_validation() {
         let cases = [
             vector_public_key_validation_1(),
-            vector_public_key_validation_2(),
+            //vector_public_key_validation_2(), // y out of range
```

Comment:
using this test vector, pk is invalid because of `y` invalid bytes length. Using compressed pk make the pk valid.